addons/web/static/src/views/list/list_renderer.js

```diff
@@ -711,7 +711,7 @@ export class ListRenderer extends Component {
         const { widget, attrs } = column;
         const field = this.props.list.fields[column.name];
         const aggregateValue = group.aggregates[column.name];
-        if (aggregateValue === false) {
+        if (field.type === "monetary" && aggregateValue === false) {
```

Comment:
is it correct? Can't the aggregate for monetary fields be false if the sum is 0, like it is for float?